build.sbt

```diff
@@ -348,9 +348,12 @@ def mimaPrevious(moduleName: String, scalaVer: String, ver: String, includeCats1
   lazy val excludedVersions: List[String] = List()
 
   // Safety Net for Inclusions
-  lazy val extraVersions: List[String] = List("1.0.1", "1.1.0", "1.2.0", "1.3.1", "1.4.0", "1.5.0", "1.6.1")
+  lazy val extraCats1Versions: List[String] = List("1.0.1", "1.1.0", "1.2.0", "1.3.1", "1.4.0", "1.5.0", "1.6.1")
```

Comment:
Btw checking so old versions doesn't cover new definitions (added since).
IMO we should check against the latest previous version.

Reply:
We are, these are just extras :) in sbt try `show coreJVM/mimaPreviousArtifacts` to see the list.

Reply:
Oh ok I see 👍 - why do we add the extras? To be extra sure?

Reply:
/shrug :)